server/pyproject.toml

```diff
@@ -50,9 +47,7 @@ sqlalchemy = "^1.4.44"
 pymysql = "^1.0.2"
 psycopg2 = "^2.9.5"
 pymongo = ">=4.2.0"
-redshift-connector = "^2.0.909"
 sqlalchemy-redshift = ">=0.8.11,<1"
-google-cloud-bigquery = {version = "^3.3.0", extras = ["pandas", "bqstorage"]}
```

Comment:
oohhh, good catch, we were not using those libs in the codebase ?

Reply:
Well we're using them in  the tests, but they're pulled in by toucan-connectors anyway so :shrug: